education/models/record.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (C) 2018-Today: Odoo Community Association (OCA)
+# Copyright C)  2018-Today: Pesol (<http://pesol.es>)
+# @author: Angel Moya (angel.moya@pesol.es)
+# @author: Luis Adan Jimenez (luis.jimenez@pesol.es)
+# @author: Adrián Gómez (adrian.gomez@pesol.es)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import models, fields, api, _
+
+
+class EducationRecord(models.Model):
+    _name = 'education.record'
+    _rec_name = 'code'
+
+    code = fields.Char(
+        string='Code', required=True, default=lambda self: _('New'))
+    student_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Student', required=True)
+    course_id = fields.Many2one(
+        comodel_name='education.course',
+        string='Course',
+        required=True)
+    enrollment_ids = fields.One2many(
+        comodel_name='education.enrollment',
+        inverse_name='record_id',
+        string='Enrollments')
+    record_subject_ids = fields.One2many(
+        comodel_name='education.record.subject',
+        inverse_name='record_id',
+        string='Subjects Records')
+
+    @api.model
+    def create(self, vals):
+        if vals.get('code', 'New') == 'New':
+            vals['code'] = self.env['ir.sequence'].next_by_code(
+                'education.record') or 'New'
+        return super(EducationRecord, self).create(vals)
+
+
+class EducationRecordSubject(models.Model):
+    _name = 'education.record.subject'
+    _rec_name = 'subject_id'
```

Comment:
Kindly add the _description tag, and docstring to help understand the importance of the model